content/lessons/chapter-7/mempool-transaction-1.tsx

```diff
@@ -339,6 +345,15 @@ def run():
   const handleSelectLanguage = (language: string) => {
     setLanguage(language)
   }
+  const [tooltipVisible, setTooltipVisible] = useState(false)
```

Comment:
lets make them separate useStates

```suggestion
const [tooltipVisibleOne, setTooltipVisibleOne] = useState(false)
const [tooltipVisibleTwo, setTooltipVisibleTwo] = useState(false)
```